src/assets/customLoader/ParseableLoader.tsx

```diff
@@ -0,0 +1,23 @@
+import { MantineProvider, Loader } from '@mantine/core';
+import ParseableAnimate from './ParseableAnimate';
+
+// This is a custom Parseable Icon with loading animation
+function ParseableLoader() {
+	return (
+		<MantineProvider
+			theme={{
+				components: {
+					Loader: Loader.extend({
+						defaultProps: {
+							loaders: { ...Loader.defaultLoaders, ring: ParseableAnimate },
+							type: 'ring',
```

Comment:
Is `ring` a custom loader name ? Or are you just trying to override the defaults?

Reply:
Working on that

Reply:
Changed.. Not overriding the ring loader